src/clojure/dialog/logger.clj

```diff
@@ -93,29 +93,37 @@
 
 (defn blocked?
   "True if the logger is blocked by configuration, otherwise false."
-  [logger]
-  (boolean
-    (and (string? logger)
-         (some #(prefixed? logger %)
-               (:blocked config)))))
+  ([logger]
+   (blocked? (:blocked config) logger))
+  ([levels logger]
+   (boolean
+     (and (string? logger)
+          (some #(prefixed? logger %)
+                levels)))))
```

Comment:
Rather than `levels` I would call this `blocked`, since it's a set of logger name prefixes. That said, this two-argument version is actually not specific to blocked loggers at all; it would be cleaner to pull this out as a `(some-prefixed? prefixes logger)` function that can be used internally here.

Reply:
done.